src/Resources/contao/helper/functions.php

```diff
@@ -15,9 +15,9 @@
  * @param string $strMessage
  * @param string $strLog
  */
-function log_message($strMessage, $strLog='error.log')
```

Comment:
I suppose they fall back to the standard error handler. But our method is not (only) related to errors.